styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+box-sizing: border-box;
+}  
+
+body {
+margin: 0;
+background-color: #95CDED;
+color: #ffffff;
+display: flex;
+}
+
+aside {
+	width: 35%;
+	
+}
+/*
+img {
+	max-height: 500px;
+	max-width: 500px;
+}
+*/
+
+nav {
+	background-color: #95CDED;
+	color: #ffffff;
+	max-width:; 100%;
+	display: flex;
+	justify-content: space-between;
+	padding: 10px;
+	
+			
+}
+
+a {
+	text-decoration: none;
+	color: #211CF8;
+}
+
+nav a {
+	border: 2px solid #ffffff;
+	border-radius: 5px;
+	padding: 3px;
+	
+}
+
+.second-column {
+	background color: #95CDED;
+	color: #ffffff;
+	max-width: 65%;
+		
+	
+}
+
+
+
+.portfolio {
+	background-color: #95CDED;
+	color: #ffffff;
+}
+
+
+
+
+
+.narrow-nav {
+	display: none;
+
+}
+@media screen and (max-width: 900px) {
+   
+	@media screen and (max-width: 900px) {
+	  .wide-nav {
+	    display: none;
+	  }
+
+	  .narrow-nav {
```

Comment:
this button probably needs more styling, and the position on the page doesn't feel quite right yet (unless this is in line with your wireframe - I don't have it)

Reply:
looks good and in a good position now! you might want to add some padding or margin so it is in line with your wide-nav element.